docs/your_first_grid.md

```diff
@@ -175,6 +325,77 @@ sylius_grid:
                         method: mySupplierGridQuery
 ```
 
+</details>
+
+<details open><summary>PHP</summary>
+
+```php
+<?php
+// config/packages/sylius_grid.php
+
+use App\Entity\Suplier;
+use Sylius\Bundle\GridBundle\Builder\GridBuilder;
+use Sylius\Bundle\GridBundle\Builder\Filter\BooleanFilter;
+use Sylius\Bundle\GridBundle\Builder\Filter\StringFilter;
+use Sylius\Bundle\GridBundle\Config\GridConfig;
+
+return static function (GridConfig $grid) {
+    $grid->addGrid(GridBuilder::create('app_admin_supplier', Suplier::class)
+        ->addFilter(
+            StringFilter::create('name')
+        )
+        ->addFilter(
+            BooleanFilter::create('enabled')
+        )
+    )
+};
+```
+
+OR
+
+```php
+<?php
+# src/Grid/AdminSupplierGrid.php
+
+declare(strict_types=1);
+
+namespace App\Grid;
+
+use App\Entity\Suplier;
+use Sylius\Bundle\GridBundle\Builder\Filter\BooleanFilter;
+use Sylius\Bundle\GridBundle\Builder\Filter\StringFilter;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+final class AdminSupplierGrid extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public static function getName(): string
+    {
+           return 'app_admin_supplier';
+    }
+
+    public function buildGrid(GridBuilderInterface $gridBuilder): void
+    {
+        $gridBuilder
+            ->addFilter(
+                StringFilter::create('name')
+            )
+            ->addFilter(
+                BooleanFilter::create('enabled')
+            )
+        ;    
+    }
+    
+    public function getResourceClass(): string
+    {
+        return Suplier::class;
+    }
+}
+```
+
+</details>
```

Comment:
I feel like this does not match the YAML config above. Am I wrong ?

Reply:
You're right, done.